### ISSUE TITLE
genie parser "show environment" missing temperature readings

### ISSUE THREAD

**Issue by bskine** [CiscoTestAutomation/genieparser#788]
Newish to pyats/genie testing but mid python/netmiko/nornir enthusiast.  Trying to run a parser to retrieve temps from a fleet of ISR4K routers (IOSXE 17.6.3a)... "show env all" ( which is on the genie parser list) returns 
`Traceback (most recent call last):
  File "/mnt/c/Users/pboynto/PycharmProjects/pyats/parse/env.py", line 10, in <module>
    results = single_device.parse(cmd)
  File "src/genie/conf/base/device.py", line 531, in genie.conf.base.device.Device.parse
  File "src/genie/conf/base/device.py", line 570, in genie.conf.base.device.Device._get_parser_output
  File "src/genie/conf/base/device.py", line 568, in genie.conf.base.device.Device._get_parser_output
  File "src/genie/metaparser/_metaparser.py", line 329, in genie.metaparser._metaparser.MetaParser.parse
  File "src/genie/metaparser/_metaparser.py", line 322, in genie.metaparser._metaparser.MetaParser.parse
  File "src/genie/metaparser/util/schemaengine.py", line 233, in genie.metaparser.util.schemaengine.Schema.validate
genie.metaparser.util.exceptions.SchemaEmptyParserError: Parser Output is empty
`
No biggie, I decided to try just "sh env"... This does return formatted data, but is missing the temp stats for some reason???

genie output:
`{
  "_exclude": [],
  "critical_larams": 0,
  "major_alarms": 5,
  "minor_alarms": 0,
  "slot": {
    "P2": {
      "sensor": {
        "P: pwr": {
          "reading": "33    Watts",
          "state": "Normal"
        }
      }
    },
    "R0": {
      "sensor": {
        "I: 12v": {
          "reading": "5     A",
          "state": "Normal"
        },
        "P: pwr": {
          "reading": "58    Watts",
          "state": "Normal"
        },
        "V: 0.75v": {
          "reading": "765   mV",
          "state": "Normal"
        },
        "V: 1.0v": {
          "reading": "999   mV",
          "state": "Normal"
        },
        "V: 1.0v CPU": {
          "reading": "991   mV",
          "state": "Normal"
        },
        "V: 1.1v": {
          "reading": "1072  mV",
          "state": "Normal"
        },
        "V: 1.2v": {
          "reading": "1198  mV",
          "state": "Normal"
        },
        "V: 1.35v CPU": {
          "reading": "1351  mV",
          "state": "Normal"
        },
        "V: 1.5v": {
          "reading": "1514  mV",
          "state": "Normal"
        },
        "V: 1.8v": {
          "reading": "1822  mV",
          "state": "Normal"
        },
        "V: 1.8v CPU": {
          "reading": "1831  mV",
          "state": "Normal"
        },
        "V: 12v": {
          "reading": "11909 mV",
          "state": "Normal"
        },
        "V: 2.5v": {
          "reading": "2491  mV",
          "state": "Normal"
        },
        "V: 3.0v": {
          "reading": "2997  mV",
          "state": "Normal"
        },
        "V: 3.3v": {
          "reading": "3322  mV",
          "state": "Normal"
        },
        "V: 3.3v STBY": {
          "reading": "3383  mV",
          "state": "Normal"
        },
        "V: 5v": {
          "reading": "5059  mV",
          "state": "Normal"
        },
        "V: DDR": {
          "reading": "1535  mV",
          "state": "Normal"
        },
        "V: VCC CPU": {
          "reading": "860   mV",
          "state": "Normal"
        },
        "V: VNN CPU": {
          "reading": "977   mV",
          "state": "Normal"
        }
      }
    }
  }
}`

And  the cli output:
`rtr1#sh env

Number of Critical alarms:  1
Number of Major alarms:     4
Number of Minor alarms:     0

 Slot        Sensor          Current State   Reading        Threshold(Minor,Major,Critical,Shutdown)
 ----------  --------------  --------------- ------------   ---------------------------------------
 P2          RPM: fan0       Warning         25800 RPM          na
 P2          RPM: fan1       Warning         25320 RPM          na
 P2          RPM: fan2       Warning         25920 RPM          na
 P2          RPM: fan3       Warning         25500 RPM          na
 P2          P: pwr          Normal          34    Watts        na
 R0          Temp: Inlet     Critical        47    Celsius      (na ,na ,47 ,na )(Celsius)
 R0          Temp: Internal  Normal          47    Celsius      (na ,na ,na ,na )(Celsius)
 R0          Temp: Outlet    Normal          50    Celsius      (na ,na ,75 ,na )(Celsius)
 R0          Temp: CPU       Normal          53    Celsius      (na ,na ,100,na )(Celsius)
 R0          V: 12v          Normal          11917 mV           na
 R0          V: 5v           Normal          5059  mV           na
 R0          V: 3.3v         Normal          3323  mV           na
 R0          V: 3.0v         Normal          2995  mV           na
 R0          V: 2.5v         Normal          2493  mV           na
 R0          V: 1.35v CPU    Normal          1351  mV           na
 R0          V: 1.8v         Normal          1822  mV           na
 R0          V: 1.2v         Normal          1198  mV           na
 R0          V: VNN CPU      Normal          977   mV           na
 R0          V: 1.1v         Normal          1073  mV           na
 R0          V: 1.0v         Normal          999   mV           na
 R0          V: 1.8v CPU     Normal          1832  mV           na
 R0          V: DDR          Normal          1535  mV           na
 R0          V: 3.3v STBY    Normal          3384  mV           na
 R0          V: 1.5v         Normal          1514  mV           na
 R0          V: 1.0v CPU     Normal          991   mV           na
 R0          V: VCC CPU      Normal          858   mV           na
 R0          V: 0.75v        Normal          763   mV           na
 R0          I: 12v          Normal          6     A            na
 R0          P: pwr          Normal          62    Watts        na
`

**Comment by bskine** [comment]
sorry... formatting game was weak, let me try again

![carbon (4)](https://github.com/CiscoTestAutomation/genieparser/assets/55518255/51bde5b9-6c79-4c38-adee-3f6b72e1d63f)
![carbon (5)](https://github.com/CiscoTestAutomation/genieparser/assets/55518255/f2d32cd8-343e-4245-b63a-76e9f5800432)

**Comment by SohanTirpude** [comment]
Hello @bskine,

Looks like there was no support added for temperature readings. Kindly let me check on this.

Thank you.

**Comment by sowmyadn010501** [comment]
Hello bskine,
[PR](https://wwwin-github.cisco.com/pyATS/genieparser/pull/2994) has been merged. It will reflect in next release. Hence I am closing this ticket
